set.mm

```diff
@@ -469149,6 +469232,208 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Field Extensions
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c /FldExt /FinExt /AlgExt [:] $.
+
+  $( Syntax for the field extension relation. $)
+  cfldext $a class /FldExt $.
+
+  $( Syntax for the finite field extension relation. $)
+  cfinext $a class /FinExt $.
+
+  $( Syntax for the algebraic field extension relation. $)
+  calgext $a class /AlgExt $.
+
+  $( Syntax for the field extension degree operation. $)
+  cextdg $a class [:] $.
+
+  ${
+    $d e f $.
+    $( Definition of the field extension relation.  (Contributed by Thierry
+       Arnoux, 29-Jul-2023.) $)
+    df-fldext $a |- /FldExt = { <. e , f >. | ( ( e e. Field /\ f e. Field )
+      /\ ( f = ( e |`s ( Base ` f ) ) /\ ( Base ` f ) e. ( SubRing ` e ) ) ) }
+      $.
+
+    $( Definition of the field extension degree operation.  (Contributed by
+       Thierry Arnoux, 29-Jul-2023.) $)
+    df-extdg $a |- [:] = ( e e. _V , f e. ( /FldExt " { e } )
+      |-> ( dim ` ( ( subringAlg ` e ) ` ( Base ` f ) ) ) ) $.
+
+    $( Definition of the finite extension relation.  (Contributed by Thierry
```

Comment:
"field" is missing between "finite" and "extension"

Reply:
Yes, thanks!